src/SymfonyCache/TagSubscriber.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+namespace FOS\HttpCache\SymfonyCache;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use FOS\HttpCache\Tag\ManagerInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\HeaderBag;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Response;
+use FOS\HttpCache\ProxyClient\Symfony;
+
+/**
+ * Adds tag invalidation capabilities to the Symfony HTTP cache
+ *
+ * @author Daniel Leech <daniel@dantleech.com>
+ * 
+ * {@inheritdoc}
+ */
+class TagSubscriber implements EventSubscriberInterface
+{
+    /**
+     * @var ManagerInterface
+     */
+    private $manager;
+
+    /**
+     * @param ManagerInterface $manager
+     * @param mixed $options
+     */
+    public function __construct(ManagerInterface $manager)
+    {
+        $this->manager = $manager;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public static function getSubscribedEvents()
+    {
+        return [
+            Events::POST_HANDLE => 'postHandle',
+        ];
+    }
+
+    /**
+     * Check to see if the response contains tags which should be associated
+     * with the cached page.
+     *
+     * @param CacheEvent
+     */
+    public function postHandle(CacheEvent $event)
+    {
+        $response = $event->getResponse();
+
+        if (false === $response->headers->has(Symfony::HTTP_HEADER_TAGS)) {
+            return;
+        }
+
+        $this->tagResponse($response);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    private function tagResponse(Response $response)
+    {
+        $contentDigest = $this->getContentDigestFromHeaders($response->headers);
+        $tags = $this->getTagsFromHeaders($response->headers);
+        $expiry = $this->getExpiryFromResponse($response);
+        $this->manager->tagCacheId($tags, $contentDigest, $expiry);
+    }
+
+    /**
+     * Return the content digest from the headers.
+     * The content digest should be set by the Symfony HTTP cache before
+     * this method is invoked.
```

Comment:
AFAIK it is not configurable, it is by default. But I said `should` above because I am not currently sure. Will have to have a closer look.